README.md

```diff
@@ -8,34 +8,118 @@ A GitHub Action to setup [LocalStack](https://github.com/localstack/localstack)
 - Configuring the [LocalStack CLI](https://docs.localstack.cloud/get-started/#localstack-cli) to launch the Docker container with an optional API token for pro usage.
 - Installing [LocalStack AWS CLI](https://github.com/localstack/awscli-local), a thin wrapper around the `aws` command line interface for use with LocalStack to run integration tests over AWS services.
+- Export/import [LocalStack state](https://docs.localstack.cloud/user-guide/state-management/export-import-state/) as an artifact
+- Save/load [LocalStack Cloud Pods](https://docs.localstack.cloud/user-guide/state-management/cloud-pods/)
+- Start/stop a [LocalStack Ephemeral Instance](https://docs.localstack.cloud/user-guide/cloud-sandbox/application-previews/) _(EXPERIMENTAL)_
 
 ## Usage
 
-To get started, you can use this minimal example:
+### Get started minimal example
 
 ```yml
 - name: Start LocalStack
-  uses: LocalStack/setup-localstack@v0.1.2
+  uses: LocalStack/setup-localstack@v0.2.0
   with:
     image-tag: 'latest'
     install-awslocal: 'true'
   env:
     LOCALSTACK_API_KEY: ${{ secrets.LOCALSTACK_API_KEY }}
 ```
+> **NOTE**: The `LOCALSTACK_API_KEY` environment variable is required to be set if `use-pro` is set to `true`.  
+If the key is not found LocalStack by default falls back to the CE edition and displays a warning.
 
-### Inputs
+### Install only CLIs and startup later
+```yml
+- name: Install LocalStack CLIs
+  uses: LocalStack/setup-localstack@v0.2.0
+  with:
+    skip-startup: 'true'
+    install-awslocal: 'true'
+
+...
+
+- name: Start LocalStack
+  uses: LocalStack/setup-localstack@v0.2.0
+  with:
+    image-tag: 'latest'
+  env:
+    LOCALSTACK_API_KEY: ${{ secrets.LOCALSTACK_API_KEY }}
+```
+
+### Save a state later on in the pipeline
+```yml
+- name: Save LocalStack State
+  uses: LocalStack/setup-localstack@v0.2.0
+  with:
+    install-awslocal: 'true'
+    skip-startup: 'true'
+    state-backend: cloud-pods
+    state-action: save
+    state-name: my-cloud-pod
+  env:
+    LOCALSTACK_API_KEY: ${{ secrets.LOCALSTACK_API_KEY }}
+```
+> **NOTE**: The `LOCALSTACK_API_KEY` environment variable is required to be set to save/load LocalStack's state either as a Cloud Pod or an artifact.
+
+### Load an already saved state
+```yml
+- name: Start LocalStack and Load State
+  uses: LocalStack/setup-localstack@v0.2.0
+  with:
+    install-awslocal: 'true'
+    state-backend: cloud-pods
+    state-action: load
+    state-name: my-cloud-pod
+  env:
+    LOCALSTACK_API_KEY: ${{ secrets.LOCALSTACK_API_KEY }}
+```
+> **NOTE**: To load a **local state** from a different GitHub Actions workflow, one must set the `WORKFLOW_ID` environment variable.
+
+> **NOTE**: The `LOCALSTACK_API_KEY` environment variable is required to be set to **save/load** LocalStack's state either as a Cloud Pod or an artifact.
+
+### Manage App Preview
+```yml
+uses: LocalStack/setup-localstack@$v0.2.0
+  with:
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+      state-backend: preview
```

Comment:
as mentioned before, this should preferably read ephemeral - but we can also tackle that in a future iteration

Reply:
See above.